crates/viewer/re_viewer/src/open_url.rs

```diff
@@ -558,7 +558,7 @@ impl ViewerOpenUrl {
                     url_parameters.first().open_description()
                 } else {
                     ViewerOpenUrlDescription {
-                        category: "Open several URLs",
+                        category: "Several URLs",
```

Comment:
I think this saying something like "Open N URLs" would be cleaner, but a bit out of scope of this PR maybe 😅

Reply:
eh yeah maybe I shouldn't have put this into this PR, very very very offtopic. Guilty!
It actually works out fine because we always also print the extra info below alongside :)